contracts/src/v0.8/automation/dev/v2_2/AutomationRegistryBase2_2.sol

```diff
@@ -824,7 +796,7 @@ abstract contract AutomationRegistryBase2_2 is ConfirmedOwner, ExecutionPreventi
     bytes32[3] calldata reportContext,
     bytes calldata report,
     bytes32[] calldata rs,
-    bytes32[] calldata ss,
+    bytes32[] memory ss,
```

Comment:
why this change?

Reply:
ditto, explained in another comment. will try to change back.